nemo/collections/common/parts/adapter_modules.py

```diff
@@ -61,6 +61,13 @@ def get_default_strategy_config(self) -> 'dataclass':
         """
         return adapter_mixin_strategies.ResidualAddAdapterStrategyConfig()
 
+    def adapter_unfreeze(self,):
+        """
+        Sets the requires grad for all parameters in the adapter to True
```

Comment:
Hmm if this is intended to be for adapters, let's do two things - 
1) add the equivalent freeze_adapter()
2) add docstring that this should be overriden for custom adapter freezing and unfreezing instead of neural_module.freeze/unfreeze.

Reply:
sg!